exporter/datadogexporter/metrics_translator.go

```diff
@@ -33,15 +33,16 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/ttlmap"
 )
 
-// getTags maps a stringMap into a slice of Datadog tags
-func getTags(labels pdata.StringMap) []string {
+// getTags maps an attributeMap into a slice of Datadog tags
+func getTags(labels pdata.AttributeMap) []string {
 	tags := make([]string, 0, labels.Len())
-	labels.Range(func(key string, value string) bool {
-		if value == "" {
+	labels.Range(func(key string, value pdata.AttributeValue) bool {
+		v := value.StringVal() // TODO(codeboten): Fix as part of https://github.com/open-telemetry/opentelemetry-collector/issues/3815
```

Comment:
Should we add this to open-telemetry/opentelemetry-collector#3815 as part of a task list?